tap-agent/src/agent/sender_allocation.rs

```diff
@@ -187,6 +187,20 @@ impl Actor for SenderAllocation {
             allocation_id = %state.allocation_id,
             "Closing SenderAllocation, triggering last rav",
         );
+        loop {
+            match state.refresh_all_database().await {
+                Ok(value) => {
+                    state.unaggregated_fees = value;
+                    break;
+                }
+                Err(err) => {
+                    error!(
+                        error = %err,
+                        "There was an error while calculating last unaggregated receipts. Retrying in 30 seconds...");
```

Comment:
Would there be more detail in the logs than this to help investigate? This doesn't tell us what the error was, just where it happened.

Reply:
Also, grammar-wise, did you mean to say, "There was an error while calculating **_the_** last unaggregated receipts. Retrying in 30 seconds..."? I'm not sure how to interpret what this error is telling me.

Reply:
The details are printed in line 198.

Reply:
Grammar-wise, I don't have strong opnions. gonna update it.

Reply:
done